template/core/data-test/src/main/resources/bookList-Success.json

```diff
@@ -0,0 +1,50 @@
+{
+  "numFound": 1518,
+  "start": 0,
+  "numFoundExact": true,
+  "docs": [
+    {
+      "key": "1",
```

Comment:
Same key could cause potential issues when rendering the UI if we use it as a view key.